cluster-provision/gocli/opts/aaq/aaq.go

```diff
@@ -0,0 +1,60 @@
+package aaq
+
+import (
+	"embed"
+	"fmt"
+	"regexp"
+
+	"github.com/sirupsen/logrus"
+	k8s "kubevirt.io/kubevirtci/cluster-provision/gocli/pkg/k8s"
+	"kubevirt.io/kubevirtci/cluster-provision/gocli/pkg/libssh"
+)
+
+//go:embed manifests/*
+var f embed.FS
```

Comment:
For the ones that use a small number of files thats alot better indeed.
But for one that has many files i think it would look a bit unappealing to have 10+ variables initialized at the start of the file. for example.. the kubevirt provider PR introduces a provision k8s opt. which depends on 21 files

what do you think ?